manifests/server/default_privileges.pp

```diff
@@ -0,0 +1,144 @@
+define postgresql::server::default_privileges (
```

Comment:
Could you please add the summary and parameter details  which will help with autogenerating REFERENCE.md file prior to each release.
For example
https://github.com/puppetlabs/puppetlabs-postgresql/blob/main/manifests/server/database_grant.pp#L1
https://github.com/puppetlabs/puppetlabs-postgresql/blob/main/REFERENCE.md

Reply:
Of course, i'll do it this week.

Reply:
Thanks alot @mtancoigne for incorporating the comments

Reply:
It's done.